packages/lib/CalendarService.ts

```diff
@@ -293,7 +293,7 @@ export default abstract class BaseCalendarService implements Calendar {
           currentStart = dayjs(currentEvent.startDate.toJSDate());
 
           if (currentStart.isBetween(start, end) === true) {
-            return events.push({
+            events.push({
```

Comment:
The `return` here stopped the loop what was supposed to accumulate multiple instances of this recurring event, instead limiting it to a single instance (the first in-range). I think it was just a typo.

The loop still terminates correctly, since whe while condition checks the date range and has a safety limit of maxIterations.